CSS/documentation.css

```diff
@@ -119,15 +150,37 @@ main{
     font-size: 1.6rem;
 }
 
+/************************** avatar component documentation styles ********************/
 
+.component-display{
+    /* border : 3px solid red; */
+    display : flex;
+    flex-direction: row;
+    align-items : center;
+    justify-content : center;
+    flex-wrap: wrap;
+    gap : 2rem;
+    padding : 1.5rem;
+    background-color : rgba(102, 51, 153, 0.356);
```

Comment:
Try to remain consistent when using colors. Use variables , this will increase code readability